src/main/java/it/smartcommunitylab/aac/webauthn/persistence/WebAuthnUserAccount.java

```diff
@@ -0,0 +1,135 @@
+package it.smartcommunitylab.aac.webauthn.persistence;
+
+import java.util.Set;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+import javax.validation.constraints.NotNull;
+
+import com.yubico.webauthn.data.ByteArray;
+
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import it.smartcommunitylab.aac.SystemKeys;
+import it.smartcommunitylab.aac.core.model.UserAccount;
+
+@Entity
+@Table(name = "webauthn_users", uniqueConstraints = @UniqueConstraint(columnNames = { "provider_id", "username" }))
+@EntityListeners(AuditingEntityListener.class)
+public class WebAuthnUserAccount implements UserAccount {
+
+    private static final long serialVersionUID = SystemKeys.AAC_CORE_SERIAL_VERSION;
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    // TODO: civts, use converter
+    @Column(unique = true)
+    private String userHandle;
+    @OneToMany(mappedBy = "parentAccount", fetch = FetchType.LAZY, orphanRemoval = true)
+    private Set<WebAuthnCredential> credentials;
+
+    @NotNull
+    @Column(name = "subject_id")
+    private String subject;
+    private String emailAddress;
+    private String realm;
+    private String username;
+
+    @Column(name = "provider_id")
+    private String provider;
+
+    public Long getId() {
+        return id;
+    }
+
+    public ByteArray getUserHandle() {
```

Comment:
getter/setters here should just handle the field type. remove

Reply:
Fixed in be249b9178c90134b9bc131733a36115d019992b